qpid-dispatch/openshift/deploy-amq-interconnect-1-basic-to-basic.yml

```diff
@@ -0,0 +1,10 @@
+- hosts: localhost
+
+  roles:
+    - role: ansible-interconnect-image
+      openshift_template_name: "amq-interconnect-1-basic-to-basic"
+      openshift_template_url: "https://raw.githubusercontent.com/rh-messaging-qe/iqa_topologies/dlenoch-openshift2/qpid-dispatch/openshift/templates/interconnect-1-basic-to-basic.yml"
```

Comment:
@enkeys I believe we will need a PR after this gets merged to adjust the branch name in the URL.

Reply:
I did it after merging.